Gemfile.lock

```diff
@@ -478,7 +475,7 @@ GEM
       rspec-support (~> 3.10.0)
     rspec-instafail (1.0.0)
       rspec
-    rspec-mocks (3.10.1)
+    rspec-mocks (3.10.2)
```

Comment:
version bump necessary to fix this issue: https://github.com/rspec/rspec-mocks/issues/ 1396